### PR TITLE
CAMEL-11820: Upgrade optaplanner to version 7.3.0.Final

### PR DESCRIPTION
**Pull request by dmvolod**
@ge0ffrey Geoffrey, could you please a look at these changes?

### CODE REVIEW

**Comment by davsclaus** [comment]
Thanks for the PR it has been merged. Do you mind closing this?

**Comment by dmvolod** [comment]
PR closed, thank you.

File: parent/pom.xml [hunk 1]
```diff
@@ -689,6 +689,7 @@
     <twilio-version>7.9.1</twilio-version>
     <twitter4j-bundle-version>4.0.6_1</twitter4j-bundle-version>
     <twitter4j-version>4.0.6</twitter4j-version>
+    <uberfire-maven-support-version>1.3.0.Final</uberfire-maven-support-version>
```
Comment (ge0ffrey) [inline parent/pom.xml]: optaplanner engine doesn't use uberfire, but this can't hurt either (optaplanner workbench does, but that's not used by camel)

**Comment by ge0ffrey** [comment]
@dmvolod  Looks good, thanks for PRing this and thanks david for merging it :)

**Pull request closed without merging**